crates/ecolor/src/color32.rs

```diff
@@ -111,20 +112,19 @@ impl Color32 {
     pub fn from_rgba_unmultiplied(r: u8, g: u8, b: u8, a: u8) -> Self {
```

Comment:
should we document that this performs pre-multiplication in gamma space? 🤔

Reply:
I think we should properly specify and document this on the Color32 type level. Going to leave a longer comment at the PR top level.